packages/join-block/src/Settings.php

```diff
@@ -97,6 +99,10 @@ public static function init()
             Field::make('text', self::IDEAL_POSTCODES . '_api_key', 'Ideal Postcodes API Key'),
             Field::make('text', self::GET_ADDRESS_IO . '_api_key', 'getAddress.io API Key'),
 
+            Field::make('separator', 'mailchimp', 'Mailchimp'),
+            Field::make('text', 'mailchimp_api_key', 'Mailchimp API key'),
```

Comment:
In both cases, can we have some documentation here, either help text or something more detailed with a link to MailChimp documentation that explains how these can be found and added? For example, I am pretty sure I could locate the API key, but I've not heard of this Audience ID thing, so I wouldn't be able to find it easily.